converters/fromZigbee.js

```diff
@@ -8639,6 +8639,33 @@ const converters = {
             return result;
         },
     },
+    yale_lock_operation_event: {
```

Comment:
Move this converter to `yale.js` `fzLocal` (like: https://github.com/Koenkk/zigbee-herdsman-converters/blob/82990cd19b72847f98f4d90ad4d462a12c9c8cba/devices/tuya.js#L291)